web-common/src/main/resources/META-INF/resources/view/login.jsp

```diff
@@ -19,6 +19,9 @@
 <body>
 <%@ include file="header.jsp" %>
 <div id="loginContainer" class="container">
+    <div class="confidential-message text-center" > 
```

Comment:
Please use text instead of images because text will easier to maintain in case we need to make changes and to localize between different languages

Reply:
Done